### PR TITLE
Creation of posts

### PR DESCRIPTION
**Pull request by ktotam1**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -87,6 +87,7 @@ dependencies {
     implementation "androidx.room:room-runtime:$room_version"
     annotationProcessor "androidx.room:room-compiler:$room_version"
     testImplementation "androidx.room:room-testing:$room_version"
+    apply plugin: 'com.google.gms.google-services'
```
Comment (hamzaremmal) [inline app/build.gradle:90]: why did you add this plugin ?

File: app/src/main/.metals/metals.mv.db
Comment (hamzaremmal) [inline app/src/main/.metals/metals.mv.db:1]: Can you remove this file also, unnecessary

File: build.gradle [hunk 2]
```diff
@@ -32,4 +33,6 @@ subprojects {
 //        google()  // Google's Maven repository
 //        mavenCentral()  // Maven Central repository
 //    }
-//}
\ No newline at end of file
+//}
+
+apply plugin: 'com.google.gms.google-services'
```
Comment (hamzaremmal) [inline build.gradle:38]: same here, why do we need to enable the plugin at the root of the project ?

File: app/build.gradle (hunk 1 above)
Comment (ktotam1) [reply app/build.gradle:90]: was added in order to build with a bad android-settings.json, but now that the issue is resolved will remove

**Review by hamzaremmal — APPROVED** [review]
